app.py

```diff
@@ -1,103 +1,101 @@
 #!/usr/bin/env python
-"""main app file for modeler"""
+"""Main app file for modeler."""
 
-import psycopg2
+import os
+import sys
 import time
-from pyspark import SparkConf
-from pyspark import sql as pysql
+import os.path
+import argparse
```

Comment:
a pep8 testing tool might complain about the ordering here, they should be alphabetical